cfpb/consumerfinance.gov
# Update ccdb5-ui@2.4.12

## Changes

- Update ccdb5-ui@2.4.12


## How to test this PR

1. Pull branch and run `./frontend.sh`
2. Check that the menu header on http://localhost:8000/data-research/consumer-complaints/search and see that the links have necessary space and no unnecessary underlines.
